src/ray/protobuf/serve.proto

```diff
@@ -27,28 +27,30 @@ option java_multiple_files = true;
 message AutoscalingConfig {
```

Comment:
it is not good practice (not allowed) to change the number here, the number is used to make binary data compatible for serialization and deserialization between versions. 
https://developers.google.com/protocol-buffers/docs/proto3#assigning_field_numbers

Reply:
I didn't know about this– thanks for the info! I've restored the original field numbers.